site/docs/remote-caching.md

```diff
@@ -306,6 +307,31 @@ You may want to delete content from the cache to:
 * Create a clean cache after a cache was poisoned
 * Reduce the amount of storage used by deleting old outputs
 
+## Local action cache
+
+Bazel supports directory based action cache. It may be very useful, when
```

Comment:
I rephrased your words a bit to fit in better with the rest of the document:

Bazel can use a directory on the filesystem as a remote cache. This is
useful for sharing build artifacts when switching branches and/or working
on multiple workspaces of the same project i.e. multiple checkouts. Bazel
does not garbage collect the directory and thus one might want to set up a
cron job or similar to periodically clean up the directory. The disk cache
can be enabled with the following flags.

```
build --experimental_remote_spawn_cache
build --disk_cache=/path/to/disk/cache
``` 

Additionally, one can pass a user specific path to the `--disk_cache` flag
via the `~` character. Bazel will replace this with the current user's home
directory. This comes in handy when one wants to enable the disk cache for
all developers of a project via the project's checked in `.bazelrc` file.